astropy/io/fits/tests/test_header.py

```diff
@@ -1384,8 +1384,32 @@ def test_header_insert_before_keyword(self):
         assert list(header.keys())[-3] == 'TEST1'
 
     def test_remove(self):
-        # TODO: Test the Header.remove() method; add support for ignore_missing
-        pass
+        header = fits.Header([('A', 'B'), ('C', 'D')])
+
+        #When keyword is present in the header it should be removed.
```

Comment:
This is a really minor nitpick, but please put a space between `#` and the text of a comment.